### PR TITLE
chore(wdio): port host-attr-override suite to wdio

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
Porting over the `host-attr-override` test suite! This is a pretty straightforward one.

GitHub Issue Number: #5458 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Testing

If CI is good, we're good!

### CODE REVIEW

**Review by christian-bromann — APPROVED** [review]

File: test/wdio/host-attr-override/host-attr-override.test.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <>
+          <host-attr-override class="override"></host-attr-override>
+          <host-attr-override class="with-role" role="another-role"></host-attr-override>
+        </>
+      ),
+    });
+  });
+
+  it('should merge class set in HTML with that on the Host', async () => {
+    expect($('.default.override')).toExist();
```
Comment (rwaskiewicz) [inline test/wdio/host-attr-override/host-attr-override.test.tsx:17]:
```suggestion
    await expect($('.default.override')).toExist();
```

File: test/wdio/host-attr-override/host-attr-override.test.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  });
+
+  it('should override non-class attributes', async () => {
+    expect($('.with-role')).toHaveAttribute('role', 'another-role');
```
Comment (rwaskiewicz) [inline test/wdio/host-attr-override/host-attr-override.test.tsx:21]:
```suggestion
    await expect($('.with-role')).toHaveAttribute('role', 'another-role');
```

**Review by rwaskiewicz — CHANGES_REQUESTED** [review]

File: test/wdio/host-attr-override/host-attr-override.test.tsx (hunk 1 above)
Comment (rwaskiewicz) [reply test/wdio/host-attr-override/host-attr-override.test.tsx:17]: Unless this auto-retries? 🤔

Comment (christian-bromann) [reply test/wdio/host-attr-override/host-attr-override.test.tsx:17]: nope, it can't without `await`, good catch 👍

**Review by rwaskiewicz — APPROVED** [review]

**Comment by alicewriteswrongs** [comment]
I opened https://github.com/ionic-team/stencil/pull/5530 so I can't forget to `await` any longer!

**Pull request merged**